strings/src/main/java/org/odk/collect/strings/localization/LocalizedApplication.kt

```diff
@@ -9,7 +9,7 @@ interface LocalizedApplication {
     val locale: Locale
 }
 
-fun Context.getLocalizedString(stringId: Int, vararg formatArgs: Any): String {
+fun Context.getLocalizedString(stringId: Int, vararg formatArgs: Any?): String {
```

Comment:
Are you sure you need this change?

Reply:
I set this first to nullable because that was the way this argument was set in some of the methods in the Page class, but as I checked now the getString method should not take null as formatArgs, so I changed it to non-nullable.